net.wz2100.wz2100.yaml

```diff
@@ -0,0 +1,82 @@
+app-id: net.wz2100.wz2100
+runtime: org.kde.Platform
+runtime-version: '5.11'
+sdk: org.kde.Sdk
+command: warzone2100
+separate-locales: false
+finish-args: 
+  - --socket=wayland
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --persist=.warzone2100-3.2
+  - --socket=pulseaudio
+  - --share=network
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /share/gtk-doc
+  - /share/man
+  - /share/pkgconfig
+  - '*.a'
+  - '*.la'
+  
+modules:
+  - shared-modules/glu/glu-9.0.0.json
+  - shared-modules/glew/glew.json
+        
+  - name: physfs
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+    - -DCMAKE_INSTALL_PREFIX=/app
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DPHYSFS_BUILD_TEST=OFF
+    sources:
+      - type: archive
+        url: 'http://icculus.org/physfs/downloads/physfs-3.0.1.tar.bz2'
+        sha1: 9959a7acad0aa30d1a86d3a418e61155b0ac533c
+      
+  - name: ffmpeg
+    config-opts:
+      - --disable-everything
+      - --enable-gpl
+      - --disable-static
+      - --enable-shared
+      - --disable-doc
+      - --disable-ffplay
+      - --disable-ffprobe
+    post-install:
+      - chrpath -d /app/bin/ffmpeg
+    cleanup:
+      - /share/ffmpeg/examples
+    sources:
+      - type: archive
+        url: https://ffmpeg.org/releases/ffmpeg-4.0.2.tar.xz
+        sha256: a95c0cc9eb990e94031d2183f2e6e444cc61c99f6f182d1575c433d62afb2f97
+        
+  - name: fribidi
+    buildsystem: meson
+    config-opts:
+    - -Ddocs=false
+    sources:
+      - type: git
+        url: https://github.com/fribidi/fribidi
+        tag: v1.0.5
+        
+  - name: wz2100
```

Comment:
this needs a reconfigure like so: https://github.com/flathub/shared-modules/blob/08e9b9eead7a95139be5d1fbc24bea730fb78f7a/SDL/SDL_net-1.2.8.json (remove the existing configure and generate it on the host) to fix the aarch64 build https://flathub.org/builds/#/builders/37/builds/288/steps/5/logs/stdio since the file was generated with a toolchain that doesn't know about 64 bit ARM.